packages/api/src/did/did-doc.ts

```diff
@@ -0,0 +1,40 @@
+import { z } from 'zod'
+
+export const verificationMethod = z.object({
```

Comment:
hmmm we already have something very similar this over in `@atproto/identity` - we may want to just use the code there instead of duplicating it over to the api package 🤔

Reply:
or if the client doesn't agree with the dependencies of that package, porting just these types & verification methods over to `@atproto/common-web`